pytorch/inference/docker/1.5.1/py3/Dockerfile.cpu

```diff
@@ -87,7 +87,13 @@ RUN pip install --no-cache-dir "sagemaker-pytorch-inference<2"
 RUN curl https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.5.1/license.txt -o /license.txt
 
 RUN conda install -y -c conda-forge pyyaml==5.3.1
-RUN pip install pillow==7.1.0 "awscli<2"
+
+# Packages installed to resolve safety vulnerabilities
```

Comment:
nit: I don't think this comment is applicable to all packages (i.e. awscli < 2)

Reply:
Will change this. Thanks!